bookshelf/php.ini

```diff
@@ -0,0 +1 @@
+extension=mongodb.so
```

Comment:
Curious - what's the downside to just including mongo/redis/mysql in our default php.ini?  I kind of have this goal where we don't need to override the php.ini for bookshelf :)  I don't know, maybe it's a good idea to include it just to show folks how it works.  Thoughts?